src/core_ocean/mode_init/Registry_lock_exchange.xml

```diff
@@ -7,6 +7,22 @@
 			description="Depth of the bottom of the ocean in the lock exchange test case."
 			possible_values="Any positive real value greater than 0."
 		/>
+		<nml_option name="config_lock_exchange_low_tracer1" type="real" default_value="0.0" units=""
+			description="Concentration of debug tracer1 in one half of the domain."
+			possible_values="Any real number"
+		/>
+		<nml_option name="config_lock_exchange_high_tracer1" type="real" default_value="10.0" units=""
```

Comment:
Does having a default value >0 alter the behavior of the original test case?

Reply:
Ok. I see now that the debug tracers are not active by default. Are you planning to post a comparison between the debugTracer behavior and the sedimentTracers behavior, perhaps in the compass PR?

Reply:
@cbegeman yes I've submitted a separate PR to compass https://github.com/MPAS-Dev/compass/pull/87